docs/template/public/less/_kss-content.less

```diff
@@ -0,0 +1,206 @@
+// shared styles across pages 
+#kss-content {
+    .sectionSpacing {
+        padding-bottom: 30px;
+        padding-top: 30px;
+    }
+    .table-behavoir-cell .seperator {
+        margin: 0 50px 0 0;
+    }
+
+    /* patterns */
+    .row.visual-code {
+        margin: 0;
+    }
+
+    h3.popover-title {
+        margin-bottom: 0;
+    }
+
+    .usercard-info p {
+        margin-bottom: 0;
+    }
+
+    .list-group-item.deleteable .list-group-item-delete {
+        top: 18%;
+    }
+
+    .row {
+        margin-right: -8px;
+        margin-left: -8px;
+    }
+
+    .modal-dialog,.popover {
+        z-index: 0;
+    }
+
+    .circle {
+        border: 1px solid #cfd8e3;
+        box-shadow: 0px 1px 1px #fff;
+        margin: 0 auto 20px auto;
+        width: 100%;
+        height: 0;
+        padding: 50% 0;
+        border-radius: 50%;
+        
+    }
+
+    .circle-type {
+        font-size: 74px;
+        display: inline-block;
+        margin-top: -35%;
+    }
+
+    .seperator {
+        margin: 40px 0 80px 0;
+        position: relative;
+
+        .hexagon {
+            top: 31px;
+            left: 0;
+            position: relative;
+            width: 36px;
+            height: 20.78px;
+            background-color: #d8dfe8;
+            margin: auto;
+            display: block;
+            z-index: 2;
+
+            .glyphicon {
+                font-size: 20px;
+                position: absolute;
+                top: 10px;
+                left: 10px;
+                color: #7e858d;
+            }
+
+            .code-open {
+                font-size: 12px;
+                position: absolute;
+                top: 1px;
+                left: 6px;
+                font-weight: 900;
+                color: #7e858d;
+                
+            }
+
+            .code-close {
+                font-size: 20px;
+                position: absolute;
+                top: -2.5px;
+                left: 8px;
+                font-weight: 500;
+                color: #fff;
+                display: none;
+            }
+        }
+        .hexagon:before, .hexagon:after {
+            content: "";
+            position: absolute;
+            width: 0;
+            border-left: 18px solid transparent;
+            border-right: 18px solid transparent;
+        }
+
+        .hexagon:before {
+            bottom: 100%;
+            border-bottom: 10.39px solid #d8dfe8;
+        }
+
+        .hexagon:after {
+            top: 100%;
+            width: 0;
+            border-top: 10.39px solid #d8dfe8;
+        }
+        
+        .open-hexagon:before {
```

Comment:
Please confirm that the  open-hexagon is no longer used and if not remove this code